### ISSUE TITLE
About the TrimmedMAE loss

### ISSUE THREAD

**Issue by mangdian** [intel-isl/MiDaS#75]
Hi, again, thanks for your wonderful work.

I find that when drop out the top 20% error, the model tends to give wrong prediction on the sky area, which should have infinite depth (or zero disparity). I give two group of images as examples below.  Have you met this problem? How did you solve it? Or do you have any idea why this would happen and how to solve it ?   Thanks a lot!

![image](https://user-images.githubusercontent.com/18326098/102569703-e4e67700-4120-11eb-9794-d9a2afe656b2.png)

**Comment by mangdian** [comment]
@ranftlr

**Comment by frischzenger** [comment]
which learning rate do you use? thanks

**Comment by CJCHEN1230** [comment]
@Tord-Zhang  Hi, I'm recently dive into this topic. However, due to the lack of training code, I have absolutely no way to train results similar to the author's. Could you provide your training code? That would be really really helpful to me.
